src/overlays/actors/ovl_En_S_Goro/z_en_s_goro.c

```diff
@@ -235,16 +235,16 @@ u16 EnSGoro_ShrineGoron_NextTextId(EnSGoro* this, PlayState* play) {
 
     switch (EN_S_GORO_GET_MAIN_TYPE(&this->actor)) {
         case EN_S_GORO_TYPE_SHRINE_WINTER_A:
-            if (!(gSaveContext.save.weekEventReg[22] & 4)) {
+            if (!(CHECK_WEEKEVENTREG(WEEKEVENTREG_22_04))) {
```

Comment:
```suggestion
            if (!CHECK_WEEKEVENTREG(WEEKEVENTREG_22_04)) {
```
There are quite a few of these extra parens in this file.